drivers/ieee802154/Kconfig

```diff
@@ -0,0 +1,260 @@
+# Nordic Semiconductor nRF5 802.15.4 extended (FEM, Antenna Diversity, Coex)
+# configuration options
+
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if NRF_802154_RADIO_DRIVER
+
+menu "IEEE 802.15.4 nRF5 Radio Driver configuration"
+
+config IEEE802154_NRF5_CFG_PRE_PRIO
+	int "Radio driver pre-init configuration priority"
+	default 75
+	help
+	  Set the configuration priority number. Must be lower than radio driver
+	  initialization priority. Do not mess with it unless you know what you
+	  are doing.
+
+config IEEE802154_NRF5_CFG_POST_PRIO
+	int "Radio driver post-init configuration priority"
+	default 90
+	help
+	  Set the configuration priority number. Must be higher than radio driver
+	  initialization priority. Do not mess with it unless you know what you
+	  are doing.
+
+choice IEEE802154_NRF5_ANT_DIVERSITY_MODE
+	prompt "Antenna Diversity mode"
+	default IEEE802154_NRF5_ANT_DIVERSITY_MODE_DISABLED
+	help
+	  Select Antenna Diversity mode
+
+config IEEE802154_NRF5_ANT_DIVERSITY_MODE_DISABLED
+	bool "Antenna Diversity disabled"
+
+config IEEE802154_NRF5_ANT_DIVERSITY_MODE_MANUAL
+	bool "Antenna Diversity manual"
+
+config IEEE802154_NRF5_ANT_DIVERSITY_MODE_AUTO
+	bool "Antenna Diversity automatic"
+
+endchoice
+
+if !IEEE802154_NRF5_ANT_DIVERSITY_MODE_DISABLED
+
+menu "Antenna Diversity configuration"
+
+config IEEE802154_NRF5_ANT_DIVERSITY_PIN
+	int "Antenna selection pin"
+	default 20
+	help
+	  Pin used for antenna selection.
+
+config IEEE802154_NRF5_ANT_DIVERSITY_TOGGLE_TIME
+	int "Toggle time"
+	default 13
```

Comment:
Currently, antenna diversity API specifies 40 microseconds as the recommended toggle time, so the default should be equal 40 IMO